docs: expand cifuzz installation path section in readme (#414)
CodeIntelligenceTesting/cifuzz: README.md

the [latest release](https://github.com/CodeIntelligenceTesting/cifuzz/releases/latest/download/cifuzz_installer_windows.exe)
and execute it.

Do not forget to add the installation's `bin` directory to your `PATH`. 

> You can find the installation locations in the [installation information guide](docs/Installation-Information.md).

Reviewer: ```suggestion
You can find additional information in our [Installation Guide](link).
```
